### PR TITLE
Migrate upgrading-elasticsearch-clients-to-opensearch

### PR DESCRIPTION
**Pull request by anelook**
There was recently added a help article on migration of elasticsearch clients to opensearch.

### CODE REVIEW

File: docs/products/opensearch/howto/upgrade-clients-to-opensearch.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+nodes, the OpenSearch project forked the open source client
+libraries. This means that you can simply switch the libraries without modifying your data
+model, or application logic.
+
+Please refer to `OpenSearch compatibility documentation <https://opensearch.org/docs/latest/clients/index/>`_ as
+the source of truth.
+
+To help you with migration, we provided some `code migration examples <https://github.com/aiven/opensearch-migration-examples>`_ in our repository.
+
+Java and Spring Boot
```
Comment (lornajane) [inline docs/products/opensearch/howto/upgrade-clients-to-opensearch.rst:48]: We might need to add Spring and Boot as words to the allow list

**Review by lornajane — CHANGES_REQUESTED** [review]
I find this article a little bit jumbled - I know it's a migration but I think we can get it a more more like a "How To" to fit the devportal content format we usually use with numbered steps:
 - Newer Elasticsearch clients don't work with OpenSearch, so you will need to switch.
 - First, pin your ElasticSearch libraries at <version>
 - Switch from that to the OpenSearch clients (using all the excellent resources we referred to)
 - Migrate from ElasticSearch to OpenSearch with your Aiven service (you can do this before or after switching the clients, and we have an excellent article for you.

Do you think we can refactor?

**Comment by anelook** [comment]
Refactored, let me know if this is closer to what you meant.

**Review by lornajane — APPROVED** [review]
Looking good! Thanks :)

**Pull request merged**